src/midi/midi_fluidsynth.cpp

```diff
@@ -212,6 +213,17 @@ static std::string find_sf_file(const std::string &name)
 	return "";
 }
 
+static void log_unknown_midi_message(const std::vector<uint8_t>& msg)
+{
+	auto append_to = [](const std::string& str, const uint8_t val) {
+		std::bitset<8> val_bits(val);
+		std::string val_str = "0b" + val_bits.to_string();
+		return str + (str.empty() ? "" : ", ") + val_str;
+	};
+	const auto msgs_str = std::accumulate(msg.begin(), msg.end(), std::string(), append_to);
+	LOG_WARNING("FSYNTH: unknown MIDI message sequence: %s", msgs_str.c_str());
```

Comment:
Hmm, maybe rather log it as hex values (e.g. `0f 23 01`)?

Reply:
added